Automattic/wp-calypso
# Atomic: Add blacklisted plugins from April 2019 and May 2019

#### Changes proposed in this Pull Request

- Add list of blocked plugins from April and May of 2019. 
- These plugins are not compatible on the Atomic environment.
- Details: p9F6qB-2I5-p2

List of plugins blocked:

- powered-cache
- broken-link-checker
- mycred
- userpro
- clearfy
- cornerstone
- facetwp-manipulator
- popup-builder
- under-construction-page
- wp-optimize-by-xtraffic
- wp-phpmyadmin-extension

#### Testing instructions



- Checkout the live branch available below
- Visit the plugins page for an Atomic site
- Search for the blocked plugins (can be seen on the `Files changed`) list of this PR
- Attempt to install it
- Note that the installation does not go through
